examples/QIR/Development/Main.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Microsoft.Quantum.Sdk.Tools
+{
+    public static class DefaultEntryPoint
+    {
+        private static void Main(string[] args) =>
+            Console.WriteLine(
+                "Full support for executing projects compiled to QIR is not yet integrated into the Sdk. " +
+                "For more information about the feature, see https://github.com/microsoft/qsharp-compiler/tree/main/src/QsCompiler/QirGeneration. " +
+                "The generated QIR can be executed by manually linking the QIR runtime. " +
+                "See https://github.com/microsoft/qsharp-runtime/tree/main/src/QirRuntime for further instructions.");
```

Comment:
```suggestion
                "See https://github.com/microsoft/qsharp-runtime/tree/main/src/Qir/Runtime for further instructions.");
```
I just noticed that this path is wrong now that Cesar and I changed the folder structure. We should update that in the main.cs the compiler generates as well.

Reply:
Thanks for catching this - I think I actually need to update two more places.